src/buttons/index.ts

```diff
@@ -0,0 +1,60 @@
+import './index.scss';
+
+export const Buttons = (root: HTMLElement): void => {
+  const content = `
+  <div class="buttons-container">
+    <div>
+      <div class="custom-operation">
+        <button class="button operators">(</button>
+        <button class="button operators">)</button>
+        <button class="button memory-operators">MC</button>
+        <button class="button memory-operators">M+</button>
+        <button class="button memory-operators">M-</button>
+        <button class="button memory-operators">MR</button>
+        <button class="button unary-operators">2<div class="pow">nd</div></button>
+        <button class="button unary-operators">x<div class="pow">2</div></button>
+        <button class="button unary-operators">x<div class="pow">3</div></button>
+        <button class="button operators" value="**">^</button>
+        <button class="button unary-operators">e<div class="pow">x</div></button>
+        <button class="button unary-operators">10<div class="pow">x</div></button>
+        <button class="button unary-operators">1/x</button>
+        <button class="button unary-operators"><div class="sqrt">2</div>√x</button>
+        <button class="button unary-operators"><div class="sqrt">3</div>√x</button>
+        <button class="button operators"><div class="sqrt">y</div>√x</button>
```

Comment:
you need to separate button view to different files and generate it via config file